app/apps/server/converters/messages.js

```diff
@@ -78,7 +78,7 @@ export class AppMessagesConverter {
 	}
 
 	convertAppMessage(message) {
-		if (!message) {
+		if (!message || !message.room) {
```

Comment:
Can you please describe the motivation for this change? It may be better to validate this somewhere else 🤔

Reply:
yes, I wrote an app to test it before. I found if the message doesn't contain the room property will cause the server to crash.

Reply:
https://github.com/RocketChat/Rocket.Chat/blob/6a101befc94b2310d797db8df343ee78160f7e4f/app/apps/server/converters/messages.js#L85 Here we use the `room.id` directly. If the room is `undefined`, an undefined error will be thrown.

Reply:
Right, I was confused that his might be a business validation that is out of the responsibility of the converter, but since it impacts directly the function, then it's in the right place :)